lldb/source/Plugins/ExpressionParser/Clang/ClangExpressionParser.cpp

```diff
@@ -352,6 +353,73 @@ static void SetupDefaultClangDiagnostics(CompilerInstance &compiler) {
   }
 }
 
+// NOTE: should be kept in sync with sdkSupportsBuiltinModules in
+// Toolchains/Darwin.cpp
+static bool
+sdkSupportsBuiltinModulesImpl(const llvm::Triple &triple,
```

Comment:
I'm about to change the implementation of this function in the Darwin driver

Reply:
Implementation in what sense? Changes to the SDK version checks?

Reply:
Posted https://github.com/llvm/llvm-project/pull/102239 for it.

Reply:
Ah yes I accounted for that in this patch already